docs/Shopify.md

```diff
@@ -4,6 +4,14 @@ Introducing BTCPay Server for Shopify – open-source payment gateway that enabl
 
 Our integration with Shopify allows you connect your self-hosted BTCPay Server with your [Shopify store](https://www.shopify.com/), enabling you accept Bitcoin payments swiftly and securely.
 
+:::warning
+
+⚠️ Shopify V1 Integration Deprecated
+
+New users can no longer use the Shopify V1 integration with BTCPay Server. Please refer to the [Shopify V2 integration guide](ShopifyV2/README.md) for instructions on setting up BTCPay Server with Shopify. 
```

Comment:
nitpick: the link to `ShopifyV2/README.md` here works but maybe do just `/ShopifyV2/` in case we later replace it with a dedicated page and not import it? Or does this not work for some reason?